rules/prevent-abbreviations.js

```diff
@@ -96,6 +96,10 @@ const getNameReplacements = (name, options, limit = 3) => {
 		return {total: 0};
 	}
 
+	if (name.includes('i18n')) {
```

Comment:
It should not be hard-coded directly here. There should be a list like `allowList` that is checked in the above if-statement.

Reply:
Done!
I added `i18n` and `l10n` to `defaultAllowList`.

But we should probably move it to `defaultReplacements`, because something like `const i18nData = {}` will not work, and it will still suggests to refactor it to `index18nData` but that's wrong.

What do you think ? :smile:

Reply:
It's known https://github.com/sindresorhus/eslint-plugin-unicorn/issues/752, you'll have to use `ignore` option for now.

Reply:
Right, so I added `defaultIgnore`, should we also, add a new option, `extendDefaultIgnore` ?

Reply:
> Right, so I added defaultIgnore, should we also, add a new option, extendDefaultIgnore ?

Nah. We can add it when it's needed. I see no reason to not want these ignores.